acceptance-tests/src/test/java/io/blueocean/ath/offline/pipeline/DownstreamLinkTest.java

```diff
@@ -25,21 +29,47 @@
     @Rule
     public SSEClientRule sseClient;
 
+    @Rule
+    public TestName testName = new TestName();
+
     @Test
     public void downstreamJobLinkAppearsInRunResult() throws Exception {
 
-        final String upstreamJobScript = "stage ('stageName') { build 'downstreamJob' }";
-        ClassicPipeline upstreamJob = pipelineFactory.pipeline("upstreamJob").createPipeline(upstreamJobScript);
+        ClassicPipeline upstreamJob = pipelineFactory.pipeline("upstreamJob").createPipeline(getJobScript("upstream"));
+        freestyleJobFactory.pipeline("downstreamJob").create("echo blah");
+
+        upstreamJob.build();
+        sseClient.untilEvents(upstreamJob.buildsFinished);
+        sseClient.clear();
+
+        upstreamJob.getRunDetailsPipelinePage().open(1);
 
+        find("//*[contains(text(),'Triggered Builds')]").isVisible(); // Heading for table of builds
+        find("//*[contains(text(),'downstreamJob')]").isVisible(); // row pointing to downstream build
+    }
+
+    @Test
+    @Ignore("Work in progress")
+    public void sequentialStages() throws Exception {
+
+        ClassicPipeline upstreamJob = pipelineFactory.pipeline("upstreamJob").createPipeline(getJobScript("upstream"));
         freestyleJobFactory.pipeline("downstreamJob").create("echo blah");
 
+        upstreamJob.build();
+        upstreamJob.build();
         upstreamJob.build();
         sseClient.untilEvents(upstreamJob.buildsFinished);
         sseClient.clear();
 
         upstreamJob.getRunDetailsPipelinePage().open(1);
+        upstreamJob.getRunDetailsPipelinePage().open(2);
+        upstreamJob.getRunDetailsPipelinePage().open(3);
 
         find("//*[contains(text(),'Triggered Builds')]").isVisible(); // Heading for table of builds
         find("//*[contains(text(),'downstreamJob')]").isVisible(); // row pointing to downstream build
     }
+
+    private String getJobScript(String name) throws IOException {
+        return new String(IOUtils.toByteArray(getClass().getResource(DownstreamLinkTest.class.getName() + "/" + testName.getMethodName() + "." + name + ".groovy")));
```

Comment:
@Dohbedoh the problem is here :) I'm fixing that especially as 1.24.x doesn't have same level of commons-io dependencies